docs/reference/changelog.rst

```diff
@@ -2,6 +2,14 @@
 Changelog
 *********
 
+2.2.2 (2025-MM-DD)
+------------------
+
+Bug fixes:
+
+- Make sure the :ref:`uv plugin<craft_parts_uv_plugin>` is re-entrant on code
```

Comment:
Maybe more clear since `source:` is used in craft.yaml files.

```suggestion
- Make sure the :ref:`uv plugin<craft_parts_uv_plugin>` is re-entrant on source
```